src/components/LanguagePicker/LanguagePickerListSearchInput.tsx

```diff
@@ -0,0 +1,80 @@
+import React, {useState} from 'react'
+import {StyleSheet, TextInput, TouchableOpacity, View} from 'react-native'
```

Comment:
why pull directly from react native instead of the components / RNPaper?

Reply:
Thanks! I will apply next time. That component has been removed finally.